src/lib/config/index.ts

```diff
@@ -44,11 +44,22 @@ export const LOGGING = set(
 	}
 )
 
+export const POSTGRES = set(
+	process.env.POSTGRES || 'postgres://user:password@host:5433/lpg'
```

Comment:
Guessing this isn't really meant to be part of this change, but obviously `@host` won't do much.